runtime/near-vm-errors/src/lib.rs

```diff
@@ -7,6 +7,7 @@ use std::any::Any;
 use std::fmt::{self, Error, Formatter};
 use std::io;
 
+// ----------8<----------
```

Comment:
What is it?

Reply:
A marker for the delimitation of the section of stuff that should be moved to near-vm-runner (ascii art for a line with scissors mark)